libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -36,6 +36,7 @@
 #include "dnn_io_proc.h"
 #include "dnn_backend_common.h"
 #include <tensorflow/c/c_api.h>
+#include <pthread.h>
```

Comment:
#include "libavutil/thread.h"

btw, i today do not have more time to review starting from the tf backend,

Reply:
I have corrected this and handled the case where pthreads aren't supported. 
No problem, you may review whenever you find the time. I'll continue further.